### PR TITLE
Fix webdriver test, add debugging notes and test warning

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: run_test.sh [hunk 1]
```diff
@@ -121,12 +122,13 @@ function run_webdriver_test {
     echo -e "#### Running webdriver tests in parallel"
     python3 -m pytest --tests-per-worker auto webdriver_tests/*.py
   else
-    python3 -m pytest webdriver_tests/*.py
+    python3 -m pytest webdriver_tests/ranking_test.py
```
Comment (chejennifer) [inline run_test.sh:125]: change this back to `webdriver_tests/*.py`?

**Review by chejennifer — APPROVED** [review]

**Comment by chejennifer** [comment]
Thanks for the fix!

File: docs/developer_guide.md [hunk 2]
```diff
@@ -219,3 +219,19 @@ the same region.
...
+  combined with `sleep()` to give you time to inspect the page. To enter this
+  mode, comment out this line in
+  [base_test.py](../server/webdriver_tests/base_test.py):
+  ```python
+  chrome_options.add_argument('--headless')
+  ```
+
+- Another option is to save a screenshot at various points of the test:
+  ```python
+  self.driver.save_screenshot(filename
```
Comment (shifucun) [inline docs/developer_guide.md:236]: add ")"?

File: run_test.sh (hunk 1 above)
Comment (beets) [reply run_test.sh]: oops! done.

**Review by shifucun — APPROVED** [review]
thanks for fix!

File: docs/developer_guide.md (hunk 2 above)
Comment (beets) [reply docs/developer_guide.md]: thanks for the catch! done.

**Pull request merged**